Jenkinsfile

```diff
@@ -1,4 +1,10 @@
-#!/usr/bin/env groovy
-
 /* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
-buildPlugin()
+
+buildPlugin(
+  findbugs: [
```

Comment:
Think your missing archive and run here.
```groovy
boolean runFindbugs = first && params?.findbugs?.run
boolean archiveFindbugs = first && params?.findbugs?.archive
```

https://github.com/jenkins-infra/pipeline-library/blob/ec29bbe966194298de7db8b323a816986e8a8f47/vars/buildPlugin.groovy#L117-L126
```groovy
                            if (isMaven && archiveFindbugs) {
                                def fp = [pattern: params?.findbugs?.pattern ?: '**/target/findbugsXml.xml']
                                if (params?.findbugs?.unstableNewAll) {
                                    fp['unstableNewAll'] ="${params.findbugs.unstableNewAll}"
                                }
                                if (params?.findbugs?.unstableTotalAll) {
                                    fp['unstableTotalAll'] ="${params.findbugs.unstableTotalAll}"
                                }
                                findbugs(fp)
                            }
```

Reply:
I was trying to find good example for this but I couldn't. Do you have any?